### DISCUSSION TITLE
Does it make sense to convert the image to grayscale?

### DISCUSSION THREAD

**Discussion by FelipeErmeson** [towhee-io/towhee#2687] [Q&A]
Apparently the convnext_xlarge_in22k model, for example, was trained on colored images, so it would be best to leave the image colored. I also don't see any operator that transforms the image to grayscale.

**Comment by junjiejiangjjj** [comment]
Image conversion can be done using cv2, defining a custom operator, and using it in the pipeline.(https://towhee.readthedocs.io/en/latest/operator/usage.html#custom-operators)
